Added Standard tier as supported for B2C

EldertGrootenboer opened the pull request:
According to the announcement from December, B2C integration with API Management is now available in the Standard tier as well:
https://blogs.msdn.microsoft.com/apimanagement/2017/12/01/updates-to-azure-api-management-pricing/

azurecla commented:
Hi __@alexandair__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

PRmerger commented:
@alexandair : Thanks for your contribution to the Azure documentation! The author, @trinadhk, has been notified to review your proposed change.

markgalioto commented:
@Ja-Dunn, @Jim-Parker these updates from Alexandair are fine to merge. Who merges the approved updates in the public repo?

markgalioto commented:
@alexandair, thank you for your contributions!

Commit:
Added Standard tier as supported for B2C

According to the announcement from December, B2C integration with API Management is now available in the Standard tier as well: 
https://blogs.msdn.microsoft.com/apimanagement/2017/12/01/updates-to-azure-api-management-pricing/

Commit:
Merge pull request #1 from EldertGrootenboer/EldertGrootenboer-patch-1-b2c-standard-tier

Added Standard tier as supported for B2C

ktoliver committed:
add serial comma
